REBRANDING.md

```diff
@@ -0,0 +1,30 @@
+# Where did Eclipse JKube come from?
+
+This project was earlier called [Fabric8 Maven Plugin](https://github.com/fabric8io/fabric8-maven-plugin) which was an essential component of [Fabric8](http://fabric8.io/) project which was an end to end development platform for building, testing and deploying your applications using it's pipelines. However, project was not successful and it was finally archived. However, there were still some projects inside that org which were still relevant and made sense in as general purpose tool/library. I have provided more background in this [RedHat Developer blogpost](https://developers.redhat.com/blog/2020/01/28/introduction-to-eclipse-jkube-java-tooling-for-kubernetes-and-red-hat-openshift/#Background)
```

Comment:
> I have provided more background in this

Either replace this with "We have provided more background on this..." or "Rohan Kumar provides more background on this..."